testsuite/MDAnalysisTests/coordinates/test_chemfiles.py

```diff
@@ -102,6 +103,39 @@ def ref():
         return ChemfilesXYZReference()
 
 
+    def test_copy(self, ref):
```

Comment:
These extra tests in chemfiles and H5MD aren't super pretty, but it's the only way I could think of having them sit within import guarded regions without making `test_copying` overly complicated.